packages/date/src/index.js

```diff
@@ -379,6 +380,17 @@ export function isInTheFuture( dateValue ) {
 	return momentObject.isAfter( now );
 }
 
+/**
+ * Get the moment object from date.
+ *
+ * @param {string} dateValue Date String or Date object in the Defined WP Timezone.
+ *
+ * @return {Moment} The Moment object.
+ */
+export function getMoment( dateValue ) {
```

Comment:
We had that in the past but we decided against exposing "moment" as an official API. The idea being at some point, we could get rid of moment as it's a very heavy module (and not so well maintained anymore).

Reply:
> We had that in the past but we decided against exposing "moment" as an official API. The idea being at some point, we could get rid of moment as it's a very heavy module (and not so well maintained anymore).

In that case we can create specific functions for tests like this one for relative time.

```js
/**
 * Get the timeago or relative time according to the WordPress settings.
 *
 * @param {string}  dateValue     Date String or Date object in the Defined WP Timezone.
 * @param {boolean} withoutSuffix Whether to remove suffix e.g "ago".
 *
 * @return {string} The duration from now.
 */
export function fromNow( dateValue, withoutSuffix ) {
	const momentObject = momentLib.tz( dateValue, WP_ZONE );

	return momentObject.fromNow( withoutSuffix );
}
```

Reply:
Is this only for test ? Is it possible to use `__experimentalGetSettings`  to test this ?

Reply:
It will be same as `isInTheFuture()` which is used for tests. Using `__experimentalGetSettings` may not server the purpose.

Reply:
`isInTheFuture` is not exposed only for tests, it's also used in other packages though.

Reply:
Since the settings mapping is available via `__experimentalGetSettings`, can we rely on it for tests?

Reply:
yes, I think so.

Reply:
I have removed the exposed moment API and updated the tests to use `__experimentalGetSettings`.